### PR TITLE
new ligate haplotypes recipe

### PR DESCRIPTION
**Pull request by hardingnj**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by hardingnj** [comment]
Hi @bioconda/core I'm a bit stuck here with why this is failing. 
Seems to be related to a travis timeout?

**Comment by johanneskoester** [comment]
I have restarted the build. Let's see what happens.

File: recipes/ligatehaplotypes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - gcc
+  run:
+    - gcc
```
Comment (bgruening) [inline recipes/ligatehaplotypes/meta.yaml]: libgcc, please look at our documentation for build c/c++ package for more information's.

**Comment by hardingnj** [comment]
Hi @bioconda/core I'm not sure why this is failing- seems to be a git parsing issue?

**Comment by johanneskoester** [comment]
@daler, this seems like it comes from your git-based filter code... Any idea?

**Comment by hardingnj** [comment]
Thanks. The restart seems to have kicked it.

**Comment by johanneskoester** [comment]
Indeed. So it was just a temporal travis issue @daler.

File: recipes/ligatehaplotypes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  md5: cd0fe75c2008999ef4c7772c075b11f0
+
+build:
+  number: 0
+  skip: True  # [osx]
+  string: boost{{CONDA_BOOST}}_{{PKG_BUILDNUM}}
+
+requirements:
+  build:
+    - libgcc
```
Comment (johanneskoester) [inline recipes/ligatehaplotypes/meta.yaml]: Put gcc here, not libgcc.

Comment (johanneskoester) [inline recipes/ligatehaplotypes/meta.yaml]: To clarify: gcc in the build section, libgcc in the run section.

**Comment by daler** [comment]
@johanneskoester phew!

@hardingnj, might want to check https://bioconda.github.io/troubleshooting.html#zlib-errors for some troubleshooting options

**Comment by hardingnj** [comment]
Thanks @daler. I'm still struggling a bit. I did read those- but I don't think those env variables are used in the makefile itself.

**Comment by johanneskoester** [comment]
We have migrated our build system from Travis CI to Circle CI. If you would like to continue working on this PR, please update with latest upstream master and reopen this PR.

**Pull request closed without merging**